src/Pages/Templates/TemplateDetails/components/Tabs/TemplateSystemsTab.tsx

```diff
@@ -0,0 +1,311 @@
+    columnSortAttributes[columnIndex]
+      ? {
+          sortBy: {
+            index: activeSortIndex,
+            direction: activeSortDirection,
+            defaultDirection: 'asc', // starting sort direction when first sorting a column. Defaults to 'asc'
+          },
+          onSort: (_event, index, direction) => {
+            setActiveSortIndex(index);
+            setActiveSortDirection(direction);
+          },
+          columnIndex,
+        }
+      : undefined;
+
+  if (isLoading) {
+    return <Loader />;
+  }
+
+  if (isError) {
+    throw error;
+  }
+
+  return (
+    <Grid className={classes.mainContainer}>
+      <InputGroup className={classes.topContainer}>
+        <Flex gap={{ default: 'gapMd' }}>
+          <InputGroupItem>
+            <TextInput
+              id='search'
+              ouiaId='name_search'
+              placeholder='Filter by name'
+              value={searchQuery}
+              onChange={(_event, value) => setSearchQuery(value)}
+            />
+            <InputGroupText id='search-icon'>
+              <SearchIcon />
+            </InputGroupText>
+          </InputGroupItem>
+          <FlexItem className={classes.ctions}>
+            <ConditionalTooltip
+              content='You do not have the required permissions to perform this action.'
+              show={!rbac?.templateWrite}
+              setDisabled
+            >
+              <Button
+                id='assignTemplateToSystems'
+                ouiaId='assign_template_to_systems'
+                variant='primary'
+                isDisabled={fetchingOrLoading}
+                onClick={() => navigate('add')}
+              >
+                Assign template to systems
+              </Button>
+            </ConditionalTooltip>
+          </FlexItem>
+          <ConditionalTooltip
+            content='You do not have the required permissions to perform this action.'
+            show={!rbac?.templateWrite}
+            setDisabled
+          >
+            <SystemsDeleteKebab
+              deleteFromSystems={deleteFromSystems}
+              deselectAll={deselectAll}
+              isDisabled={!rbac?.templateWrite}
+              selected={selected}
+            />
+          </ConditionalTooltip>
+        </Flex>
+        <Pagination
+          id='top-pagination-id'
+          widgetId='topPaginationWidgetId'
+          itemCount={total_items}
+          perPage={perPage}
+          page={page}
+          onSetPage={onSetPage}
+          isCompact
+          onPerPageSelect={onPerPageSelect}
+        />
+      </InputGroup>
+      <Hide hide={!!total_items}>
+        <Bullseye data-ouia-component-id='systems_list_page'>
+          <EmptyTableState
+            notFiltered={!debouncedSearchQuery}
+            clearFilters={() => setSearchQuery('')}
+            itemName='associated systems'
+            notFilteredBody='To get started, add a system to this template.'
```

Comment:
looking at this [thread](https://redhat-internal.slack.com/archives/C03G5FWK91V/p1720207618787559), i think maybe this wording should be `add this template to a system` ?

Reply:
Yes! 100% it should be.